### PR TITLE
Integration of entities

### PR DESCRIPTION
**Pull request by FloMal**
Submission for review before merge in master of activity, comment, partner, photo, product, ProductAttribute

### CODE REVIEW

File: src/AppBundle/Entity/Comment.php [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	 *
+	 * @var date
+	 * @ORM\Column (type="date")
+	 */
+	private $date;
+	
+	/**
+	 *
+	 * @var User
+	 * @ORM\ManyToOne(targetEntity="User")
```
Comment (MilaHG) [inline src/AppBundle/Entity/Comment.php]: add (inversedBy="comments")

File: src/AppBundle/Entity/Comment.php [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	 * @ORM\JoinColumn(name="user_id", referencedColumnName="id", nullable=false)
+	 */
+	private $user;
+	
+	/**
+	 *
+	 * @var Activity
+	 * @ORM\ManyToOne(targetEntity="Activity")
```
Comment (MilaHG) [inline src/AppBundle/Entity/Comment.php]: add (inversedBy="comments")

File: src/AppBundle/Entity/Partner.php [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+	 *      minMessage = "Your company name must be at least {{ limit }} characters long",
+	 *      maxMessage = "Your company name cannot be longer than {{ limit }} characters"
+	 * )
+	 */
+	private $company;
+
+	/**
+	 *
+	 * @var string
+	 * @ORM\Column (type="string", length=45)
```
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: attribut name of column is missing for firstname

File: src/AppBundle/Entity/Comment.php [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+	 *
+	 * @var string
+	 * @ORM\Column (type="text")
+	 */
+	private $comment;
+	
+	/**
+	 *
+	 * @var int
+	 * @ORM\Column (type="integer")
```
Comment (MilaHG) [inline src/AppBundle/Entity/Comment.php]: name missing

File: src/AppBundle/Entity/Partner.php [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+	 *     message = "The email '{{ value }}' is not a valid email.",
+	 *     checkMX = true
+	 * )
+	 */
+	private $email;
+
+	/**
+	 *
+	 * @var string
+	 * @ORM\Column (type="string")
```
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: attribut name of column is missing for password

File: src/AppBundle/Entity/Comment.php (hunk 1 above)
Comment (MilaHG) [inline src/AppBundle/Entity/Comment.php]: inversedBy="comments"

File: src/AppBundle/Entity/Partner.php [hunk 6]
```diff
@@ -0,0 +1,318 @@
...
+	 * 
+	 */
+	private $password;
+
+	/**
+	 *
+	 * @var string
+	 * @ORM\Column (type="string", length=14, unique=true)
```
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: attribut name of column is missing for commercialRegistry

File: src/AppBundle/Entity/Comment.php (hunk 2 above)
Comment (MilaHG) [inline src/AppBundle/Entity/Comment.php]: inversedBy="comments"
name="activity"

File: src/AppBundle/Entity/Partner.php [hunk 7]
```diff
@@ -0,0 +1,318 @@
...
+	 *      max = 14,
+	 *      exactMessage = "This value should have exactly {{ limit }} characters."
+	 * )
+	 */
+	private $commercialRegistry;
+
+	/**
+	 *
+	 * @var string
+	 * @ORM\Column (type="string", length=250)
```
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: attribut name of column is missing for address

File: src/AppBundle/Entity/Partner.php [hunk 8]
```diff
@@ -0,0 +1,318 @@
...
+	 *      minMessage = "Your address must be at least {{ limit }} characters long",
+	 *      maxMessage = "Your address cannot be longer than {{ limit }} characters"
+	 * )
+	 */
+	private $address;
+
+	/**
+	 *
+	 * @var string
+	 * @ORM\Column (type="string", length=5)
```
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: attribut name of column is missing for zip

File: src/AppBundle/Entity/Partner.php [hunk 9]
```diff
@@ -0,0 +1,318 @@
...
+	 *      max = 5,
+	 *      exactMessage = "This value should have exactly {{ limit }} characters."
+	 * )
+	 */
+	private $zip;
+
+	/**
+	 *
+	 * @var string
+	 * @ORM\Column (type="string", length=100)
```
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: attribut name of column is missing for city

File: src/AppBundle/Entity/Partner.php [hunk 10]
```diff
@@ -0,0 +1,318 @@
...
+	 *      minMessage = "The city must be at least {{ limit }} characters long",
+	 *      maxMessage = "The city cannot be longer than {{ limit }} characters"
+	 * )
+	 */
+	private $city;
+
+	/**
+	 *
+	 * @var date
+	 * @ORM\Column (type="date")
```
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: attribut name of column is missing for registryDate

File: src/AppBundle/Entity/Partner.php [hunk 11]
```diff
@@ -0,0 +1,318 @@
...
+	
+	/**
+	 *
+	 * @var ArrayCollection
+	 * @ORM\OneToMany(targetEntity="Activity", mappedBy="id_partner")
+	 */
+	private $activities;
+
+
+	public function getId() {
```
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: Add comment for all getters and setters : type of return value and  parameters.

File: src/AppBundle/Entity/Partner.php [hunk 12]
```diff
@@ -0,0 +1,318 @@
...
+	 * @ORM\Column(name="id", type="integer")
+	 * @ORM\Id
+	 * @ORM\GeneratedValue(strategy="AUTO")
+	 */
+	private $id;
+
+	/**
+	 *
+	 * @var string
+	 * @ORM\Column (type="string", length=100, unique=true)
```
Comment (MilaHG) [inline src/AppBundle/Entity/Partner.php]: name="company"

File: src/AppBundle/Entity/Partner.php (hunk 3 above)
Comment (MilaHG) [inline src/AppBundle/Entity/Partner.php]: name="firstname"

File: src/AppBundle/Entity/Photo.php [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+	 * @ORM\JoinColumn (name="activity_id", referencedColumnName="id", nullable=false)
+	 */
+	private $activity;
+
+	/**
+	 * Get id
+	 *
+	 * @return int
+	 */
+	public function getId() {
```
Comment (dia42r) [inline src/AppBundle/Entity/Photo.php]: comment is missing for all setters and getters : type of return value and parameters

File: src/AppBundle/Entity/Partner.php [hunk 14]
```diff
@@ -0,0 +1,318 @@
...
+
+/**
+ * Partner
+ *
+ * @ORM\Table(name="partner")
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\PartnerRepository")
+ * @UniqueEntity(fields="email", message="This email already exists")
+ * @UniqueEntity(fields="pseudo", message="This username is not available")
+ * @UniqueEntity(fields="company", message="This company has already an account")
+ * @UniqueEntity(fields="commercialRegistry", message="This commercial registry is already used")
```
Comment (MilaHG) [inline src/AppBundle/Entity/Partner.php]: should the fields "company" and "commercialRegistry" not be in the same line ?

File: src/AppBundle/Entity/Partner.php (hunk 6 above)
Comment (MilaHG) [inline src/AppBundle/Entity/Partner.php]: type="integer", length=9

File: src/AppBundle/Entity/Product.php [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+	 *
+	 * @var Activity
+	 * @ORM\ManyToOne (targetEntity="Activity", inversedBy="products")
+	 * @ORM\JoinColumn (name="activity_id", referencedColumnName="id", nullable=false)
+	 */
+	private $activity;
+
+	/**
+	 *
+	 * @var arrayCollection
```
Comment (dia42r) [inline src/AppBundle/Entity/Product.php]: I think it is ArrayCollection not arrayCollection

File: src/AppBundle/Entity/Product.php [hunk 16]
```diff
@@ -0,0 +1,123 @@
...
+	 * @ORM\OneToMany(targetEntity="Article", mappedBy="product")
+	 */
+	private $productAttributes;
+
+	/**
+	 * Get id
+	 *
+	 * @return int
+	 */
+	public function getId() {
```
Comment (dia42r) [inline src/AppBundle/Entity/Product.php]: missing comment for getters and setters : type of return value and parameters

File: src/AppBundle/Entity/ProductAttribute.php [hunk 17]
```diff
@@ -0,0 +1,102 @@
...
+	 * @var string 
+	 * @ORM\Column(name="value", type="string")
+	 * @Assert\NotBlank()
+	 */
+	private $value;
+	
+	/**
+	 *
+	 * @var Product
+	 * @ORM\ManyToOne(targetEntity="Product")
```
Comment (dia42r) [inline src/AppBundle/Entity/ProductAttribute.php]: missing inversedBy attribute

File: src/AppBundle/Entity/ProductAttribute.php [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+	private $extraFee;
+
+
+	/**
+	 * Get id
+	 *
+	 * @return int
+	 */
+	
+	public function getId() {
```
Comment (dia42r) [inline src/AppBundle/Entity/ProductAttribute.php]: comments missing for getters and setters

File: src/AppBundle/Entity/Comment.php [hunk 19]
```diff
@@ -0,0 +1,122 @@
...
+	 * @ORM\Column(name="id", type="integer")
+	 * @ORM\Id
+	 * @ORM\GeneratedValue(strategy="AUTO")
+	 */
+	private $id;
+
+	/**
+	 *
+	 * @var string
+	 * @ORM\Column (type="text")
```
Comment (dia42r) [inline src/AppBundle/Entity/Comment.php]: missing name attribute on Column for comment

File: src/AppBundle/Entity/Comment.php (hunk 4 above)
Comment (dia42r) [inline src/AppBundle/Entity/Comment.php]: missing name attribute on Column for note

File: src/AppBundle/Entity/Partner.php (hunk 12 above)
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: missing name attribute on Column for company

File: src/AppBundle/Entity/Partner.php (hunk 3 above)
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: missing name attribute on Column for firstname

File: src/AppBundle/Entity/Partner.php [hunk 20]
```diff
@@ -0,0 +1,318 @@
...
+	 *      minMessage = "Your contact lastname must be at least {{ limit }} characters long",
+	 *      maxMessage = "Your contact lastname cannot be longer than {{ limit }} characters"
+	 * )
+	 */
+	private $lastname;
+
+	/**
+	 *
+	 * @var string
+	 * @ORM\Column (type="string", length=45)
```
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: missing name attribute on Column for lastname

File: src/AppBundle/Entity/Partner.php (hunk 5 above)
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: missing name attribute on Column for password

File: src/AppBundle/Entity/Partner.php (hunk 6 above)
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: missing name attribute on Column for commercialRegistry

File: src/AppBundle/Entity/Partner.php (hunk 7 above)
Comment (dia42r) [inline src/AppBundle/Entity/Partner.php]: missing name attribute on Column for address

**Pull request merged**